### DISCUSSION TITLE
Audit Logs

### DISCUSSION THREAD

**Discussion by Sudarsh1010** [keizerworks/keizer-auth-api#31] [Ideas]
- **Description**: Capture and log significant actions taken within accounts and applications, such as user logins, role changes, and permissions updates.
- **Why**: Audit logs provide transparency and traceability, which is especially important for user management and security.

**Comment by aayushchugh** [comment]
Found this blog which can help us setup a basic audit system

1. https://medium.com/@alameerashraf/building-an-audit-log-system-for-a-go-application-ce131dc21394

**Reply by Sudarsh1010** [reply]
will go through this, Thanks
